### PR TITLE
[DFMD-27] Develop an E-commerce Store Database

### PR DESCRIPTION
**Pull request by MoyinoluwaA**

### CODE REVIEW

File: submissions/moyinoluwaAdenuga/database/init.sql [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+);
+
+-- Created admin info table
+CREATE TABLE admin_users (
+	id INT NOT NULL AUTO_INCREMENT,
+	admin_name VARCHAR(30) NOT NULL,
+	admin_email VARCHAR(40) NOT NULL,
+	admin_password VARCHAR(12) NOT NULL,
+	admin_phone_number VARCHAR(15) NOT NULL,
+    log_in_date DATETIME NOT NULL,
```
Comment (adaugochi) [inline submissions/moyinoluwaAdenuga/database/init.sql:18]: `created_at` is better

File: submissions/moyinoluwaAdenuga/database/init.sql [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+	id INT NOT NULL AUTO_INCREMENT,
+	product_category_id INT NOT NULL,
+	product_name VARCHAR(30) NOT NULL,
+	product_description VARCHAR(250) NOT NULL,
+	product_image BLOB(150) NOT NULL,
+	product_unit_price DECIMAL(10, 2) NOT NULL,
+	product_stock INT NOT NULL,
+	product_status VARCHAR(20) NOT NULL,
+	admin_username_id INT NOT NULL,
+	product_added_at DATETIME NOT NULL,
```
Comment (adaugochi) [inline submissions/moyinoluwaAdenuga/database/init.sql:33]: same here

File: submissions/moyinoluwaAdenuga/database/init.sql [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+    customer_firstname VARCHAR(20) NOT NULL,
+    customer_lastname VARCHAR(20) NOT NULL,
+    customer_email VARCHAR(40) NOT NULL,
+    customer_password VARCHAR(12) NOT NULL,
+    registered_at DATETIME NOT NULL,
+    PRIMARY KEY (id)
+);
+
+-- Created registered customers address table
+CREATE TABLE registered_addresses (
```
Comment (adaugochi) [inline submissions/moyinoluwaAdenuga/database/init.sql:51]: `registered_customer_addresses` is better

File: submissions/moyinoluwaAdenuga/database/init.sql [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+    zip_code VARCHAR(20),
+    country VARCHAR(30) NOT NULL,
+    phone_number VARCHAR(15) NOT NULL,
+    added_at DATETIME NOT NULL,
+    PRIMARY KEY (id),
+    FOREIGN KEY (customer_id) REFERENCES registered_customers (id)
+);
+
+-- Created customer order table
+CREATE TABLE customer_order (
```
Comment (adaugochi) [inline submissions/moyinoluwaAdenuga/database/init.sql:66]: Table name should be pluralized

File: submissions/moyinoluwaAdenuga/database/init.sql [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+	id INT NOT NULL,
+    customer_id INT NOT NULL,
+    order_amount DECIMAL(10,2) NOT NULL,
+    ordered_on DATETIME NOT NULL,
+    PRIMARY KEY (id),
+    FOREIGN KEY (customer_id) REFERENCES registered_customers (id)
+);
+
+-- Created customer order item table
+CREATE TABLE customer_item (
```
Comment (adaugochi) [inline submissions/moyinoluwaAdenuga/database/init.sql:76]: Table name should be pluralized

**Review by adaugochi — CHANGES_REQUESTED** [review]
Also, write an insert query to insert records inside these tables

**Comment by MoyinoluwaA** [comment]
@adaugochi I have made the necessary corrections. Thanks

**Review by adaugochi — APPROVED** [review]

**Pull request merged**